### PR TITLE
managing "Elastic IP" for EC2 instances

### PR DESCRIPTION
**Pull request by guessi**
sample config:

    primitive res_awseip ocf:heartbeat:awseip \
        params awscli="/usr/local/bin/aws" \
        params elastic_ip="52.9.0.123"

Signed-off-by: guessi <guessi@gmail.com>

### CODE REVIEW

**Comment by krig** [comment]
Same comment regarding `sleep 3` as in #797, other than that this looks good to me as well.

File: heartbeat/awseip [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+<shortdesc lang="en"></shortdesc>
+
+<parameters>
+
+<parameter name="awscli" unique="0">
+<longdesc lang="en">
+command line tools for aws services
+</longdesc>
+<shortdesc lang="en">aws cli tools</shortdesc>
+<content type="string" default="/usr/bin/aws" />
```
Comment (oalbrigt) [inline heartbeat/awseip]: You should define OCF_RESKEY_<parametername>_default and assign values from that in default= and to the OCF_RESKEY_<parametername> variables (usually placed near the top of the script):
http://www.linux-ha.org/doc/dev-guides/_api_definitions.html#_environment_variables

Comment (guessi) [inline heartbeat/awseip]: thank !!! I'll update later :+1:

**Comment by guessi** [comment]
@krig @oalbrigt  please review it again, thanks

I will squash them into one when it is ready.

**Comment by krig** [comment]
Looks good to me

**Pull request merged**